otterdog/eclipse-cyclonedds.jsonnet

```diff
@@ -71,6 +71,17 @@ orgs.newOrg('eclipse-cyclonedds') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('cyclonedds-insight') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      default_branch: "master",
+      delete_branch_on_merge: false,
+      private_vulnerability_reporting_enabled: true,
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
again, for new repos I would suggest to already use "read" default permissions.

Reply:
Done